### PR TITLE
Suppress unsaved warning if no memory used.  Fixes #78.

### LINKED ISSUE #78: Suppress “no save” warning if other output commands are present (or allow warning to be suppressed manually)
Currently BeebAsm outputs a warning if a source file doesn’t contain a SAVE command. This can’t be turned off.

It is common to create source files that create disc images using PUTFILE and PUTBASIC only, such as [this example](https://github.com/markmoxon/cassette-elite-beebasm/blob/master/1-source-files/main-sources/elite-disc.asm). These files always give this warning, even though they are perfectly valid.

Would it be sensible to amend the warning so it is only shown if the file doesn’t contain SAVE _or_ a PUT command? Or, alternatively, add a directive or command line flag to allow us to suppress the warning on a per-file basis?

It’s a minor thing, but it would be great not to see this error at the end of every compilation run!

### PR DESCRIPTION
**Pull request by mungre**
Also update `beebasm -h` version to v1.10rc2

### CODE REVIEW

**Comment by ZornsLemma** [comment]
Thanks for implementing this, it's a nice feature - much nicer than the tricks I've played in the past with INCBIN+SAVE in place of a PUTFILE just to avoid the warning. For what it's worth I've had a quick look at the code change and it looks good. It would be good if this can go into 1.10.

File: src/main.cpp [hunk 1]
```diff
@@ -42,7 +42,7 @@
 using namespace std;
 
 
-#define VERSION "1.10-pre"
+#define VERSION "1.10rc2"
```
Comment (chriskillpack) [inline src/main.cpp:45]: Thanks for changing this. I had just cut a v1.10rc2 before I saw this change. I'll delete that release and we will re-cut rc2 to include this PR.

**Review by chriskillpack — APPROVED** [review]
LGTM

**Comment by mungre** [comment]
Thank you.  The windows exe seems fine.

**Pull request merged**